lib/pure/asyncmacro.nim

```diff
@@ -314,7 +314,7 @@ proc asyncSingleProc(prc: NimNode): NimNode {.compileTime.} =
                          createFutureVarCompletions(futureVarIdents, nil))
     outerProcBody.add procCb
 
-    # -> return retFuture
+    # -> return internalRetFuture
```

Comment:
This is not the way Nim does comments `#-> ` it should be just `# `. Looks kinda JavaScripty.

Reply:
The arrow is indicating what the macro will generate. (Also theres a space here between `#` and `->`)  Would you leave our the arrow here, it would look like commented-out code, which it is not.